domains/fakhri

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "FDMZ17"
+    },
+    "record": {
+        "CNAME": "fdmz17.github.io",
+        "TXT": ["dh=34f63b4b3c46cd73b9247f543bcb3007262e1db0"]
```

Comment:
This is not how you add discord verification, see https://is-a.dev/docs/discord-verification/